packages/build-tools/src/ios/pod.ts

```diff
@@ -1,12 +1,40 @@
 import path from 'path';
 
+import fs from 'fs-extra';
 import { Ios } from '@expo/eas-build-job';
 import spawn from '@expo/turtle-spawn';
 
 import { BuildContext } from '../context';
 
 export async function installPods<TJob extends Ios.Job>(ctx: BuildContext<TJob>): Promise<void> {
   const iosDir = path.join(ctx.reactNativeProjectDirectory, 'ios');
+
+  if (ctx.env.EAS_BUILD_COCOAPODS_CACHE_URL) {
+    ctx.logger.info('Running using experimental CocoaPods cache proxy...');
+    const podFilePath = path.join(iosDir, 'Podfile');
+    const originalPodFileContentBuffer = await fs.readFile(podFilePath);
+    const originalPodFileContent = originalPodFileContentBuffer.toString('utf-8');
```

Comment:
```suggestion
    const originalPodfileContent = await fs.readFile(podfilePath, 'utf8');
```

Reply:
Actually, this should be "contents".

Reply:
Afaik both are correct, but anyway I fixed the suggestion 👍